webf/lib/src/css/matrix.dart

```diff
@@ -564,6 +564,10 @@ class CSSMatrix {
     // Compute rotation and renormalize matrix.
     double angle = atan2(row0y, row0x);
 
+    // fix animation transform: rotate(360deg);
```

Comment:
What's the meaning of '-2.4492935982947064e-16' ?

Reply:
Why needs to call `toString()` ？

```
if (angle == -2.4492935982947064e-16) {
}
```

won't work?

Reply:
del toString() ,work fine